### PR TITLE
Link assurance via Hugo relrefs

### PR DESCRIPTION
**Pull request by chadwcarlson**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Hugo (which we use for documentation) uses Markdown as its file format.
...
 * Always use inline links.
 * Do not hard-wrap prose text. Set your text editor to soft wrapping.
-* Internal links should be absolute (starting with `/`). Single pages (i.e. `/configuration/services/elasticsearch.md` should be linked using the `.md` suffix. List pages `/configuration/services/_index.md`, which will be served from `/configuration/services.html` should use the explicit `.html` suffix.
+* Internal links should be absolute (starting with `/`), and should use Hugo's built-in [`relref`]() shortcode syntax. Using `relref` ensures that internal links are always valid by explicitly defining the file that should be linked, and tying all internal links to the documentation's build. That is, if you write a link to an non-existent file, the build will fail. Single pages should be specified by its page (i.e. `{{< relref "/configuration/services/elasticsearch.md#some-subsection" >}}`) and list pages to that sections index file (i.e. `{{< relref "/configuration/services/php/_index.md#some-subsection" >}}`).
```
Comment (Crell) [inline CONTRIBUTING.md:19]: Seems to be missing a link to documentation?

File: docs/layouts/shortcodes/integrations.html [hunk 2]
```diff
@@ -1,20 +1,20 @@
-
+{{ $context := . }}
```
Comment (Crell) [inline docs/layouts/shortcodes/integrations.html:1]: What does this line do?

Comment (chadwcarlson) [reply docs/layouts/shortcodes/integrations.html]: redundant. Passing  `.` to relref is enough to get the context

File: CONTRIBUTING.md (hunk 1 above)
Comment (chadwcarlson) [reply CONTRIBUTING.md]: Fixed.

**Pull request merged**